azure-pipelines.yml

```diff
@@ -6,74 +6,103 @@
 trigger:
 - master
 
-pool:
-  vmImage: ubuntu-20.04
+resources:
+  repositories:
+  - repository: sonic-swss
+    type: github
+    name: Azure/sonic-swss
+    endpoint: build
 
-container:
-  image: sonicdev-microsoft.azurecr.io:443/sonic-slave-buster:latest
-  options: --privileged
+stages:
+- stage: Build
 
-steps:
-- task: DownloadPipelineArtifact@2
-  inputs:
-    source: specific
-    project: build
-    pipeline: 1
-    artifact: sonic-buildimage.vs
-    runVersion: 'latestFromBranch'
-    runBranch: 'refs/heads/master'
-  displayName: "Download artifacts from latest sonic-buildimage build"
+  jobs:
+  - job:
+    displayName: "Build"
```

Comment:
Now that this job is nested in a stage called "Build", this job is displayed as "Build Build", which doesn't make much sense. I can think of a few possible solutions to make this better:

1. Change this job's `displayName` to `Python 3` or `Python 3 Wheel` or `Python Wheel`
2. Split this into two jobs, the unit test and the wheel build and define the `displayName` fields appropriately
3. Split this into two stages, `UnitTest` and `Build` and define the `displayName` fields appropriately for the jobs

Thoughts?

Reply:
choice 1 is good.